macros/parserMultiAnswer.pl

```diff
@@ -411,22 +411,21 @@ sub setMessage {
 ######################################################################
 
 #
-#  Produce the name for a named answer blank
-#  (Use the standard name for the first one, and
-#   create the prefixed names for the rest.)
+#  Produce the name for a named answer blank.
+#  (When the singleResult option is true, use the standard name for the first
+#  one, and create the prefixed names for the rest.)
 #
 sub ANS_NAME {
-  my $self = shift; my $i = shift; my $name;
+  my $self = shift; my $i = shift;
+  $self->{answerNames} = {} if !defined($self->{answerNames});
+  return $self->{answerNames}->{$i} if defined($self->{answerNames}->{$i});
```

Comment:
The second `->` is not necessary, as it is implied after the first one.  So `$self->{answerNames}{$i}` is sufficient, and follows the style used in the rest of the file.  Although we don't have a style guide, it seems best to keep a consistent style within a single file.

Reply:
I didn't realize latter ->'s were implied.  Good to know.